packages/eas-cli/src/update/utils.ts

```diff
@@ -19,40 +21,46 @@ export type UpdateGroupDescription = FormatUpdateParameter & {
   runtimeVersion: string;
 };
 
+export type FormattedUpdateGroupDescription = {
+  message: string;
+  group: string;
+  platforms: string;
+  runtimeVersion: string;
+};
+
+export type FormattedUpdateGroupDescriptionWithBranch = FormattedUpdateGroupDescription & {
+  branch: string;
+};
+
 export const UPDATE_COLUMNS = [
   'Update message',
   'Update runtime version',
   'Update group ID',
   'Update platforms',
 ];
 
-export function getUpdateGroupsWithPlatforms<
-  UpdateFragment extends Pick<Update, 'platform' | 'group'>
->(updates: UpdateFragment[]): (UpdateFragment & { platforms: string })[] {
-  return Object.values(groupBy(updates, updates => updates.group)).map(updateGroup => ({
-    ...updateGroup[0],
-    platforms: formatPlatformForUpdateGroup(updateGroup),
-  }));
-}
+export const UPDATE_COLUMNS_WITH_BRANCH = ['Branch', ...UPDATE_COLUMNS];
 
 export function getPlatformsForGroup({
-  group,
-  updates,
+  group = '',
+  updates = [],
 }: {
-  group: string;
-  updates: { group: string; platform: string }[];
+  group: string | undefined;
+  updates: { group: string; platform: string }[] | undefined;
 }): string {
   const groupedUpdates = groupBy(updates, update => update.group);
   return formatPlatformForUpdateGroup(groupedUpdates[group]);
```

Comment:
```suggestion
  return group && formatPlatformForUpdateGroup(groupedUpdates[group]);
```
+ drop `= ''` default from `group`.

Reply:
we want this function to return `N/A` if input is invalid, so `group && ...` wouldn't work, but i inlined a ternary. this should resolve the issue.